### ISSUE TITLE
Full-tuning and Linear accuracy is strange

### ISSUE THREAD

**Issue by ssb0621** [JieShibo/PETL-ViT#8]
I'm now proceeding with a reimplementation of full-tuning and Linear accuracy.
However, unlike the paper, both full-tuning and linear have performance below 50%.
Can you tell me why?

**Comment by JieShibo** [comment]
Hi.

The results of full fine-tuning, linear probing, and VPT are borrowed from [_VPT paper_](https://arxiv.org/abs/2203.12119) and can be reproduced with [this repo](https://github.com/kmnp/vpt).

I guess the main reasons for the performance gap are that 
- _VPT paper_ uses SGD instead of AdamW for linear, and searches the base learning rate and weight decay for each single task. The optimal hyper-parameters can be found [here](https://drive.google.com/drive/folders/1ldhqkXelHDXq4bG7qpKn5YEfU6sRehJH?usp=sharing).
- _VPT paper_ does not normalize the input images, while our pipeline follows [_NOAH paper_](https://arxiv.org/abs/2206.04673﻿) and uses normalized inputs (see [here](https://github.com/JieShibo/PETL-ViT/blob/6adb8d03b9359627627a5f81ebb6e8e849590714/convpass/vtab/vtab.py#L55)).

**Comment by ssb0621** [comment]
Thanks!!!
I have a one question about optimal hyper-parameters
I know vtab is a dataset for domain adaptation
In optimal hyper-parameters.csv, cifar100 batchsize is 2,048(linear), but vtab training set has a 1,000 training sample
is it possible? 
i think that they use full training sample(cifar100 has a 50,000 dataset)

**Comment by JieShibo** [comment]
Sorry, I have no idea. This file is provided by VPT’s author @KMnP [here](https://github.com/KMnP/vpt#hyperparameters-for-experiments-in-paper). Maybe you can ask her or raise an issue in [this repo](https://github.com/kmnp/vpt).

**Comment by KMnP** [comment]
Hi @sungbinson, we set 2048 for all linear probing experiments since all the experiments including other larger datasets. In the case of VTAB-1k, setting 2048 means we use the entire training data (800 during validation, 1000 during training) for one batch. Let me know if you have other questions, or you can raise an issue in VPT repo like @JieShibo suggested.
